src/OrbitGl/TrackManager.cpp

```diff
@@ -501,7 +501,7 @@ VariableTrack* TrackManager::GetOrCreateVariableTrack(const std::string& name) {
   return track.get();
 }
 
-AsyncTrack* TrackManager::GetOrCreateAsyncTrack(const std::string& name) {
+AsyncTrack* TrackManager::GetOrCreateAsyncTrack(std::string name) {
```

Comment:
Yes, I'm planning on doing that, but not in this PR because there are plenty of subleties in here. The non-const version of std::map::operator[] doesn't work with std::string_view because the Key is of type `std::string`. So we have to replace that by a combination of `std::map::find` and `std::map::emplace`. Also it's unclear to me if the ordering matters. The first call to `std::map::operator[]` already creates an entry and it will be filled later - like I said plenty of subleties that I have to test manually that I would prefer to keep out of this PR.

BTW, I think I will opt for the version where the parameter becomes a string_view for consistency with all the other GetOrCreate functions that I refactored. The rationale is the following: On average the `Get`-path is much more likely than the `Create` path, so the string is only needed in very rare cases. For the `Get`-path a string_view is enough.